seqlen/seqlen.go

```diff
@@ -0,0 +1,78 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// seqlen filters sequences that are above a length cut-off.
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/biogo/biogo/alphabet"
+	"github.com/biogo/biogo/io/seqio"
+	"github.com/biogo/biogo/io/seqio/fasta"
+	"github.com/biogo/biogo/seq/linear"
+)
+
+// Flags assign input, output file names and minimum length cut-off to their default
```

Comment:
This is documentation for the Go standard library. I don't think we need that, but rather comments on bíogo-specific things.